### PR TITLE
Optimize poweredby page

### PR DESCRIPTION
**Pull request by haetao**
Modify some words and resort.

### CODE REVIEW

**Comment by haetao** [comment]
`上海定菱网络科技有限公司+http://www.qbaolive.com/` deleted
`别吃胖 http://www.biechipang.net/`  deleted
`龙猫直播 http://www.tvlongmao.com/`  deleted

File: community/content/poweredby/_index.cn.md [hunk 1]
```diff
@@ -48,83 +45,83 @@ chapter = true
...
-1. 丽人丽妆 http://www.lrlz.com/ 
-1. 龙猫直播 http://www.tvlongmao.com/ 
+1. 开课啦 https://www.kaike.la/
+1. 康帕斯科技 http://www.compasshz.com/ 
+1. 科大讯飞 https://www.iflytek.com/ 
+1. 客如云 https://www.keruyun.com/
+1. 丽人丽妆 https://www.lrlz.com/ 
 1. 马蜂窝 http://www.mafengwo.com/ 
 1. 每日优鲜便利购 https://www.missfresh.cn/ 
 1. 美利金融 https://www.mljr.com/
```
Comment (tuohai666) [inline community/content/poweredby/_index.cn.md]: 美利车金融

File: community/content/poweredby/_index.cn.md [hunk 2]
```diff
@@ -48,83 +45,83 @@ chapter = true
...
-1. 龙猫直播 http://www.tvlongmao.com/ 
+1. 开课啦 https://www.kaike.la/
+1. 康帕斯科技 http://www.compasshz.com/ 
+1. 科大讯飞 https://www.iflytek.com/ 
+1. 客如云 https://www.keruyun.com/
+1. 丽人丽妆 https://www.lrlz.com/ 
 1. 马蜂窝 http://www.mafengwo.com/ 
 1. 每日优鲜便利购 https://www.missfresh.cn/ 
 1. 美利金融 https://www.mljr.com/ 
 1. 秒卖房 https://www.maifang.com/
```
Comment (tuohai666) [inline community/content/poweredby/_index.cn.md]: I can't access the URL.

File: community/content/poweredby/_index.cn.md [hunk 3]
```diff
@@ -48,83 +45,83 @@ chapter = true
...
 1. 美利金融 https://www.mljr.com/ 
 1. 秒卖房 https://www.maifang.com/ 
 1. 魔力耳朵 https://magicears.com.cn/ 
-1. 男衣库 http://www.nanyiku.com/ 
+1. 男衣库 https://www.nanyiku.com/
+1. 南潮信息科技 https://ruff.io/ 
 1. 你我金融 https://www.niiwoo.com/ 
 1. 拍拍贷 https://www.ppdai.com/ 
 1. 汽车之家  https://www.autohome.com.cn/ 
 1. 权大师 http://www.quandashi.com/
```
Comment (tuohai666) [inline community/content/poweredby/_index.cn.md]: https://www.quandashi.com/

File: community/content/poweredby/_index.cn.md [hunk 4]
```diff
@@ -48,83 +45,83 @@ chapter = true
...
+1. 男衣库 https://www.nanyiku.com/
+1. 南潮信息科技 https://ruff.io/ 
 1. 你我金融 https://www.niiwoo.com/ 
 1. 拍拍贷 https://www.ppdai.com/ 
 1. 汽车之家  https://www.autohome.com.cn/ 
 1. 权大师 http://www.quandashi.com/ 
 1. 赛特斯CertusNet http://www.certusnet.com.cn/ 
-1. 定菱网络科技 http://www.qbaolive.com/ 
-1. 南潮信息科技 https://ruff.io/ 
 1. 食蚁兽网络 http://www.41soo.com
```
Comment (tuohai666) [inline community/content/poweredby/_index.cn.md]: I can't access the URL.

File: community/content/poweredby/_index.en.md [hunk 5]
```diff
@@ -7,124 +7,121 @@ chapter = true
 
 ## Register
 
-Welcome companies who have used ShardingSphere to register here, your support is our biggest encouragement.
+Welcome companies who have used ShardingSphere to register here, your support is important to us.
```
[the remark this answers is not in the document]
Comment (tuohai666) [reply community/content/poweredby/_index.en.md]: Please refer to: https://github.com/apache/incubator-shardingsphere/issues/234

File: community/content/poweredby/_index.cn.md [hunk 6]
```diff
@@ -48,83 +45,83 @@ chapter = true
...
-1. 科大讯飞 http://www.iflytek.com/ 
-1. 客如云 http://www.keruyun.com/ 
-1. 丽人丽妆 http://www.lrlz.com/ 
-1. 龙猫直播 http://www.tvlongmao.com/ 
+1. 开课啦 https://www.kaike.la/
+1. 康帕斯科技 http://www.compasshz.com/ 
+1. 科大讯飞 https://www.iflytek.com/ 
+1. 客如云 https://www.keruyun.com/
+1. 丽人丽妆 https://www.lrlz.com/ 
 1. 马蜂窝 http://www.mafengwo.com/
```
Comment (tuohai666) [inline community/content/poweredby/_index.cn.md]: https://www.mafengwo.cn/

**Pull request merged**